_pkgdown.yml

```diff
@@ -28,15 +28,15 @@ navbar:
         - text: Data Flow
           href: articles/blueprint/dataflow.html
         - text: Product Map
-          href: articles/blueprint/products_map.html
+          href: articles/blueprint/product_map.html
         - text: Features
           navbar: Features
-        - text: "`qenv`"
-          href: articles/blueprint/qenv.html
+        - text: teal_data
+          href: articles/blueprint/teal_data.html
         - text: Filter Panel
           href: articles/blueprint/filter_panel.html
-        - text: Delayed Data Loading (DDL)
-          href: articles/blueprint/ddl.html
+        - text: In-App Data
+          href: articles/blueprint/in_app_data.html
```

Comment:
Should we rearrange the order of topics and place the `In-App Data` topic after the `teal_data` topic? It seems like these two topics should be closely together.

Reply:
Good idea :+1: